Update Contribute section (#390)
AntennaPod/antennapod.github.io: _i18n/en/contribute/translate.md

Out of the estimated [world population](https://en.wikipedia.org/wiki/World_population) of 8 billion people, 'only' 1.5 billion [speak English](https://www.ethnologue.com/insights/ethnologue200/). This may not mean much per se. But if anything, we can conclude that making AntennaPod available in more languages, makes the app accessible to more people. As a side benefit, localizing the app also helps our project grow beyond the current main userbases (Germany and the US).

We therefore much appreciate your help!

> Our translations are hosted on Weblate, an open source translations platform. Anyone can sign up there and [start translating for AntennaPod](https://hosted.weblate.org/projects/antennapod). You'll find everything you need to know in [Weblate's documentation for translators](https://docs.weblate.org/en/latest/user/translating.html){:target="_blank"}.

Author: ```suggestion
Our translations are hosted on Weblate, an open source localisation platform. Anyone can sign up there and [start translating for AntennaPod](https://hosted.weblate.org/projects/antennapod). You'll find everything you need to know in [Weblate's documentation for translators](https://docs.weblate.org/en/latest/user/translating.html){:target="_blank"}.
```

<!-- mdpo-disable -->
{% capture img-translationstatus %}
{% include image.html
   alt="
       <!-- mdpo-enable-next-line -->
       Graph with status of AntennaPod translations on Weblate.

       "
   loc="https://hosted.weblate.org/widget/antennapod"
   file="horizontal-auto.svg"
   width="100%"
   islink = true
%}
{% endcapture %}

<object data="https://hosted.weblate.org/widget/antennapod/horizontal-auto.svg" type="image/svg+xml" width="100%" height="auto" crossorigin="anonymous">
<a href="https://hosted.weblate.org/engage/antennapod" target="_blank">{{- img-translationstatus | strip -}}</a>
</object>
<!-- mdpo-enable -->

## 1. The app

> The app is currently available in more than 38 languages. While there are more languages in Weblate, we only include those languages in the app which are at least 40% translated. Below that threshold the app hardly provides a good user experience.

Author: ```suggestion
The app is currently available in more than 38 languages. While there are even more translations in Weblate, we only include languages which are at least 40% translated. Below that threshold the app hardly provides a good user experience.
```

Reviewer: ```suggestion
The app is currently available in more than 38 languages. While there are even more translations in Weblate, we only include in the app languages which are at least 40% translated. Below that threshold the app hardly provides a good user experience.
```

It seems clearer to still mention "in the app".

Author: ```suggestion
The app is currently available in more than 38 languages. While there are even more translations in Weblate, we only include languages in the app which are at least 40% translated. Below that threshold the app hardly provides a good user experience.
```

Updated translations are published with every 'point release'. These are releases like 2.0, 2.5, etc., which introduce improvements and new features. On average, it might take a month or two before your work is available in a release.

If you contributed translations, you will be mentioned in the app with your Weblate username under `Settings` » `About` » `Contributors`.

## 2. The website
The website exists of four 'components', each representing a part of the website. We encourage you to follow this order:

1. **General** and **Short strings**: the first elements a user sees on the website
2. **Documentation**: our support materials, which are directly available via the app
3. **Contribute** (comes last because a basic understanding of English is needed anyway in order to contribute)

> Updated translations are published once in a while. New languages are enabled on the website only after priority 1 & 2 components are fully translated. Please let us know through the forum (see below) if those components are done. We'll then try to find a community member who can review a test version of the website, to improve the quality of translations. After the review, we can publish the new language version.

Author: ```suggestion
Updated translations are published once in a while. New languages are enabled on the website only after priority 1 & 2 components are fully translated. In other words: when everything but the contribution section is done. Please let us know through the forum (see below) if that's the case. We'll then try to find a community member who can review a test version of the website, to improve the quality of translations. After feedback is processed, we can publish the new language version.
```

Reviewer: ```suggestion
Updated translations are published once in a while. New languages are enabled on the website only after priority 1 & 2 components are fully translated. In other words: when everything but the `Contribute` section is done. Please let us know through the forum (see below) if that's the case. We'll then try to find a community member who can review a test version of the website, to improve the quality of translations. After feedback is processed, we can publish the new language version.
```

Author: ```suggestion
Updated translations are published once in a while. New languages are enabled on the website only after priority 1 & 2 components are fully translated. In other words: when everything but the 'Contribute' section is done. Please let us know through the forum (see below) if that's the case. We'll then try to find a community member who can review a test version of the website, to improve the quality of translations. After feedback is processed, we can publish the new language version.
```

## 3. The app store listings
In addition to the app and website, we also translate texts which appear in the various app stores. We encourage you to follow this order:

1. **Short description**
2. **Full description**
3. **Screenshot titles**

> We have to comply with a character limit for the short descriptions and screenshot titles. We therefore manually review improve these texts before they are pulled and made available. Please let us know on the forum once you've translated these texts.

Author: ```suggestion
We have to comply with a character limit for the short descriptions and screenshot titles. We therefore manually review and improve these texts before we publish them. Please let us know on the forum when you have completed translations of these texts.
```
